runtime/queries/dhall/textobjects.scm

```diff
@@ -0,0 +1,36 @@
+; Queries for helix to select textobjects: https://docs.helix-editor.com/usage.html#textobjects
+;;  function.inside
+;; function.around
+;; class.inside
+;; class.around
+;; test.inside
+;; test.around
+;; parameter.inside
+;; comment.inside
+;; comment.around
+
+;; Queries for navigating using textobjects
+
```

Comment:
Were these comments a reference while writing these? I think they can be cleaned up now

Reply:
sure, will do.

Reply:
(done)